Joystream-Role-Bot/src/controls/control.ts

```diff
@@ -132,32 +154,52 @@ export const setMemberRole = async (client: Client): Promise<void> => {
     //   console.error("Error removing member roles:", error);
     // }
 
-    await discordMember.roles.add(RoleAddress.membershipLinked);
 
-    const roleUpdatePromises = qnMember.roles.map(async (memberRole) => {
-      const memberRoleGroupId = memberRole.groupId;
-      const mappedRoles = roleMap[memberRoleGroupId];
+    const discordMemberAllRolesIds = discordMember.roles.cache.map((role) => role.id)
+    const roleAddressArray: string[] = Object.values(RoleAddress);
```

Comment:
What's the point of that? You can just do `RoleAddress[id]`, not need to use an array

Reply:
![Image](https://github.com/Joystream/discord-bot/assets/128012202/a393932b-6511-4392-bb65-f138c98b9b04) What this means is: